src/game/game2048.ts

```diff
@@ -27,13 +27,17 @@ export interface IGameState {
  // This is the main game class. 
  export class Game2048 {
   private scores: number = 0;
-  public grid: Grid;
+  private grid: Grid;
   private userActionsQueue: Action[] = [];
 
   constructor(size: number, private rand: IRandom) {
     this.grid = new Grid(size);
   }
 
+  public getGrid() {
+    return this.grid;
+  }
```

Comment:
Putting `grid` behind a getter is safer

Reply:
I'll start from ehre instead!

--

Comment posted using [Unblocked](https://getunblocked.com/dashboard/team/461c33ed-2d8d-468d-9409-d2cec26e4741/thread/305951ef-808e-4f06-9d8e-64f58b58af82?message=09cfa182-b592-4be0-8758-6941d151926c).

Reply:
Ok, try this from the comment section!